addons/mail/sfu_server/src/utils/utils.js

```diff
@@ -0,0 +1,164 @@
+import * as config from "#src/config.js";
+
+const ASCII = {
+    color: {
+        red: "\x1b[31m",
+        green: "\x1b[32m",
+        yellow: "\x1b[33m",
+        cyan: "\x1b[36m",
+        white: "\x1b[37m",
+        default: "\x1b[0m",
+    },
+};
+
+export class Logger {
+    /**
+     * @param {string} name
+     * @param {Object} [options]
+     * @param {string} [options.logLevel]
+     * @param {boolean} [options.timestamp]
+     * @param {boolean} [options.useColors]
+     */
+    constructor(name = "", { logLevel, timestamp, useColors } = {}) {
+        this._name = name;
+        if (!(useColors ?? config.LOG_COLOR)) {
+            this._colorize = (text) => text;
+        }
+        if (!(timestamp ?? config.LOG_TIMESTAMP)) {
+            this._getTimeStamp = () => "";
+        }
+        switch (logLevel ?? config.LOG_LEVEL) {
+            case "none":
+                this.error = () => {};
+            // eslint-disable-next-line no-fallthrough
+            case "error":
+                this.warn = () => {};
+            // eslint-disable-next-line no-fallthrough
+            case "warn":
+                this.info = () => {};
+            // eslint-disable-next-line no-fallthrough
+            case "info":
+                this.debug = () => {};
+            // eslint-disable-next-line no-fallthrough
+            case "debug":
+                this.verbose = () => {};
+        }
+    }
+    /**
+     * @returns {string} a date of this format: `yyyy-mm-dd hh:mm:ss,mmm`
+     */
+    _getTimeStamp() {
+        const now = new Date();
+        const min = now.getMinutes().toString().padStart(2, "0");
+        const sec = now.getSeconds().toString().padStart(2, "0");
+        const ms = now.getMilliseconds().toString().padStart(3, "0");
+        const month = now.getMonth() + 1;
+        return `${now.getFullYear()}-${month}-${now.getDate()} ${now.getHours()}:${min}:${sec},${ms} `;
+    }
+    /**
+     * @param {string} level formatted level, e.g. `:ERROR:`
+     * @param {string} text
+     * @param {string} [color]
+     */
+    _log(level, text, color) {
+        // eslint-disable-next-line no-console
+        console.log(
```

Comment:
I just changed it so that "error"/"warn" go to `stderr`, while the others go to `stdout, (mediasoup worker log documentation indicates that they do the same)